.github/workflows/ci.yml

```diff
@@ -124,6 +140,15 @@ jobs:
           cd sdist-test
           ../ci/fix-home bundle/build.sh "$bundle_os"
 
+      - name: "(Prerelease only) Upload bundle"
+        if: "${{ env.CI_PRERELEASE == 'true' && steps.build.outputs.do-not-prerelease != 'true' }}"
+        uses: "actions/upload-artifact@v3"
+        with:
+          name: "${{ runner.os }}-bundle"
+          path: |
+            sdist-test/bundle/*.sha
+            sdist-test/bundle/*.tar.gz
```

Comment:
The [docs for `path`](https://github.com/ncipollo/release-action#action-inputs=) state:
> An optional set of paths representing artifacts to upload to the release. This may be a single path or a comma delimited list of paths (or globs)

Since I've not used this action before, will the above work? Or does it need to be
`path: "sdist-test/bundle/*.sha,sdist-test/bundle/*.tar.gaz"`

Reply:
Wrong docs! Those are the docs for `ncipollo/release-action`, which I used on line 231; the `artifacts` property on that action is a single glob. This action is `actions/upload-artifact`, and the multi-line format for `path` is used as an example [here](https://github.com/actions/upload-artifact#upload-using-multiple-paths-and-exclusions).

Reply:
Double D'oh!